src/index.js

```diff
@@ -20,11 +20,17 @@ export default function () {
             this.currentFixtureName = this.escapeHtml(name);
         },
 
-        _renderErrors (errs) {
+        _renderErrors (testRunInfo) {
             this.report += this.indentString('<failure>\n', 4);
             this.report += this.indentString('<![CDATA[', 4);
+            
+            if (testRunInfo.unstable)
+                this.report += this.indentString('\n(unstable)\n', 6);
 
-            errs.forEach((err, idx) => {
+            if (testRunInfo.screenshotPath)
```

Comment:
If `screenshotPath` exists, you should always include it to the report, not only when a test is failed. Any test can take a screenshot using the `t.takeScreenshot` action.

Reply:
I'm feel a bit disappointed with JUnit, because there is no official schema for the format, and the format is not flexible at all. Faced with the format restrictions, I think that `<system-out>` is good enough for the screenshot path. However, I'm afraid that it will be a breaking changes for users that got used to the our old report format and whose CI systems or test report viewers can't properly display the `<system-out>` element. So I need some time to discuss with other TestCafe team members if we can make such changes in the report format.

Reply:
@AndreyBelym do you have any update on this?